src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -33,6 +33,8 @@
 using VSLangProj140;
 using OLEServiceProvider = Microsoft.VisualStudio.OLE.Interop.IServiceProvider;
 using OleInterop = Microsoft.VisualStudio.OLE.Interop;
+using Microsoft.CodeAnalysis.CSharp;
```

Comment:
This assembly is language-agnostic and cannot reference both languages. Please use an ILanguageService if necessary, but you probably already have some you can use.